src/projectile.cpp

```diff
@@ -136,6 +136,10 @@ void apply_ammo_effects( const tripoint &p, const std::set<std::string> &effects
         }
     }
 
+    if( effects.count( "ACID_DROP" ) > 0 ) {
+        g->m.add_field( p, fd_acid, 1, 0 );
+    }
```

Comment:
Use what? There's no such thing as `ACID_DROP` anywhere in the project (neither in this PR).